src/tls.rs

```diff
@@ -848,6 +848,9 @@ fn map_result_ssl(ssl: &Handshake, bssl_result: c_int) -> Result<()> {
                 // SSL_ERROR_SYSCALL
                 5 => Err(Error::TlsFail),
 
+                // SSL_ERROR_PENDING_SESSION
```

Comment:
AFAICT this is only relevant to legacy session ID-based resumption, which is forbidden in TLS 1.3 (and QUIC), so I don't think we need this (which is why it was left out originally IIRC).

But on the other hand it also doesn't hurt I guess.